firefox-ios/Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -415,8 +415,6 @@ class TabManagerImplementation: LegacyTabManager, Notifiable, WindowSimpleTabsPr
             delegates.forEach { $0.get()?.tabManager(self, didRemoveTab: tab, isRestoring: false) }
         }
         tabs = normalTabs
-
-        privateConfiguration = LegacyTabManager.makeWebViewConfig(isPrivate: true, prefs: profile.prefs)
```

Comment:
@nbhasin2 I don't have full context on this line but removing it resolves the regression (and so far at least I haven't been able to observe any problems with the app after removal). LMK if you have any thoughts/concerns.

Reply:
Finally got to it, its strange we are making a config while removing tabs